CHANGELOG.md

```diff
@@ -1,8 +1,17 @@
 # CHANGELOG
 
-## Unreleased
-
-- ...
+## 1.9.0 (2018-05-03)
+
+- Fixed undefined variable (#588)
+- Fix for exceptions throwing exceptions when setting event id (#587)
+- Fix monolog handler not accepting Throwable (#586)
+- Add `excluded_exceptions` option to exclude exceptions and their extending exceptions (#583)
```

Comment:
I would add here that is a 2.x forward compat functionality.

Reply:
I don't think we should do that yet. We don't know how and if when we release (bit of expectation management), when we do we can create the upgrade guide with all options that are forward compatible and which need removing / updating.

Notice that `excluded` is currently not documented for `1.x` and I intend to leave it that way (but ofcourse do mention it in the upgrade guide later).